app/src/ingestion/pdf_postprocess.py

```diff
@@ -114,8 +114,27 @@ def _add_link_markdown(e_text: EnrichedText) -> EnrichedText:
     return e_text
 
 
+def _add_list_markdown(
+    prev_e_text: EnrichedText | None, current_e_text: EnrichedText
+) -> EnrichedText:
+    if (
+        prev_e_text
+        and prev_e_text.type == TextType.LIST_ITEM
+        and current_e_text.type == TextType.LIST_ITEM
+    ):
+        # if sublist, indent 2 spaces
+        if "\u2022" in current_e_text.text:
+            current_e_text.text = "  - " + current_e_text.text
+        else:
+            current_e_text.text = "- " + current_e_text.text
+    elif current_e_text.type == TextType.LIST_ITEM:
+        current_e_text.text = "- " + current_e_text.text
```

Comment:
I think you need to update https://github.com/navapbc/labs-decision-support-tool/blob/771f61395ec8d69fd313e6ce851ce4d57995452b/app/src/ingestion/pdf_postprocess.py#L169 since it's expecting `    - `